source/_integrations/verisure.markdown

```diff
@@ -106,11 +106,11 @@ giid:
 
 ## 2 Factor Authentication Prerequisite
 
-Verisure added 2FA rules to Verisure mypages that aren't supported through their third party API integration. If you have 2FA enabled (which is forced as default) You might not be able to use this integration. Here is the suggested way to use deactivate 2FA if it's allowed in your region.
+Verisure added 2FA rules to Verisure mypages that aren't supported through their third party API integration. If you have 2FA enabled (which is forced as default) You might not be able to use this integration. Here is the suggested way to deactivate 2FA if it's allowed in your region.
 
 You can deactivate 2FA for your admin account and use that for Home Assistant but this isn't recommended. The steps below sets up a specific Home Assistant user and gives it restricted access.
 
-1. `Login` to Verisure mypages as your admin user and create a new admin user for Home Assistant
+1. `Login` to Verisure mypages as your admin user and create a new admin user for Home Assistant.
```

Comment:
```suggestion
1. Log in to Verisure My Pages as your admin user and create a new admin user for Home Assistant.
```

Reply:
yeah sorry about this one 🙈 
This was fixed in the latest commit